Tests/Tests.swift

```diff
@@ -86,4 +86,116 @@ class Tests: XCTestCase {
         
         XCTAssertEqual(description, "https://simple.org\n[\"Content-Type\": \"application/json\"]\n{\"parameter1\":\"value\"}")
     }
+
+    func testStringRangeConversionIncludesEmojiCluster() {
+        let string = "This is a *test* string. 🧐 👩‍👩‍👧 Added some emoji clusters *here*."
+        _ = {
+            let expected = "*here*"
+
+            let range = string.range(of: expected)!
+            let nsRange = range.nsRange(on: string)
+
+            let substring = (string as NSString).substring(with: nsRange)
+
+            XCTAssertEqual(substring, expected)
+        }()
+
+        _ = {
+            let expected = "*test*"
+
+            let range = string.range(of: expected)!
+            let nsRange = range.nsRange(on: string)
+
+            let substring = (string as NSString).substring(with: nsRange)
+
+            XCTAssertEqual(substring, expected)
+        }()
```

Comment:
I think these should probably be two separate tests with `string` pulled out into a constant - it looks like this test is also not the only place that string is used anyway